From 911d2a2fe6992633feefb3156eb076a3033dc32c Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Fri, 13 Dec 2019 09:33:03 +0800
Subject: [PATCH] arm64: dts: rockchip: remove duplicated arm-pmu in rk3368
 dtsi

The duplicated arm-pmu node is added by commit
d88bd40057f3983f97f5db98ebf7acda51091ad8

Fixes: d88bd40057f3 ("arm64: dts: rockchip: rk3368 add more ip nodes to core dts")
Change-Id: I6583345b5903ae4a3f7383b7f43399ed0cbafc0d
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index 62e289417705..451d7192303b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -401,21 +401,6 @@
 		};
 	};
 
-	arm-pmu {
-		compatible = "arm,armv8-pmuv3";
-		interrupts = <GIC_SPI 112 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 113 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 114 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 115 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 116 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 117 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 118 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 119 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-affinity = <&cpu_l0>, <&cpu_l1>, <&cpu_l2>,
-				     <&cpu_l3>, <&cpu_b0>, <&cpu_b1>,
-				     <&cpu_b2>, <&cpu_b3>;
-	};
-
 	amba {
 		compatible = "simple-bus";
 		#address-cells = <2>;
-- 
2.35.3

